content/en/install.md

```diff
@@ -3,101 +3,149 @@ title: Installation
 sidebar: false
 ---
 
-Installations methods include:
+The recommended method of installing SciPy depends on your preferred workflow.
+The common workflows can roughly be broken down into the following
+categories:
 
-- [Distributions](#distributions)
-- [pip](#pip-install)
-- [conda](#pip-install)
-- [Package Manager](#package_manager)
-- [Source](#source)
+- [Environment-based (e.g. `pip`, `conda`)](#environment-based) (the traditional workflow)
+- [Project-based (e.g. `uv`, `pixi`)](#project-based) (recommended for new users)
```

Comment:
The installations page should be for beginners, and they likely won't follow the project-based route. These are people who just want to get "import scipy" to work.

I think an explicit recommendation is a good idea, because newcomers should read this and immediately be able to say: "ah! this is how I install SciPy" without too many questions. I.e., we need to reduce the number of choices they have to make to as close to zero as possible.

If you want to be more nuanced about it, you could say, for env based: "recommended for beginners", and for project-based: "recommended for advanced users".

Reply:
> If you want to be more nuanced about it, you could say, for env based: "recommended for beginners", and for project-based: "recommended for advanced users".

see my comment below in the project-based section